config/config.go

```diff
@@ -85,10 +85,12 @@ type CacheConfig struct {
 
 // CallbackConfig is the configuration for callbacks
 type CallbackConfig struct {
-	// CallbackHost is the URL for broadcast callback registration.
-	CallbackHost string `json:"callback_host" mapstructure:"callback_host"`
-	// CallbackToken is the token for broadcast callback registration.
-	CallbackToken string `json:"callback_token" mapstructure:"callback_token"`
+	// Host is the URL for broadcast callback registration.
+	Host string `json:"callback_host" mapstructure:"callback_host"`
+	// Token is the token for broadcast callback registration.
+	Token string `json:"callback_token" mapstructure:"callback_token"`
+	// Enabled is the flag that enables callbacks.
+	Enabled bool `json:"enabled" mapstructure:"enabled"`
```

Comment:
I suppose, the json and the mapstructure names should be e.g. "host" (instead of "callback_host) ... and so on.
Because, for now, the names don't fit what is defined in example.config

Reply:
You are right, changing - thanks for spotting that 👁️